docs/docs/indexer/docker.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 7
+---
+
+# Starting Carp in container
+
+Carp can be run using `docker-compose`. It creates 3(+1 optional) services:
+
+- postgres - container running postgres database
+- cardano-node - container running cardano-node
+- carp - container build locally with carp
+
+### Backup service
+There is an optional container that can perform back of the database to an s3 bucket.
+It check every hour the cardano-node tip epoch and if it changes, an SQL backup is being generated.
+By defaul the backup service is commented out.
+
+Backup service will upload file to path:
+`s3://${S3_BUCKET}/${S3_FOLDER}/${NETWORK}/`
+
+**WARNING:**
+Starting Carp with backup service from scratch will create a dozens of backups until cardano-node fully synchronizes.
+
+
+### Environment setup
+
+For Your convenience You can create a `.env` file alongside `docker-compose.yml` file with:
+
+```
+NETWORK=<network type: mainnet or testnet>
```

Comment:
Slightly skeptical about inlining the `.env` format inside the documentation since these things tend to go stale. Not necessarily a reason to reject this PR, but it means we should create a follow-up ticket to make this part of the documentation more generic via a CodeBlock or something similar in the future (not as high of a priority though)

Reply:
Maybe I could list & describe variables that user can export?

Reply:
Also I have forgotten about .pgpass file -_-